js/content/store.js

```diff
@@ -2117,26 +2111,27 @@ class AppPageClass extends StorePageClass {
         }
 
         let workshop = document.querySelector("[href^='https://steamcommunity.com/workshop/browse']");
-        let morelikethis = document.querySelector("#recommended_block h2");
+        let greenlight = document.querySelector("[href^='https://steamcommunity.com/greenlight']");
```

Comment:
Could you give me one page where there is a Greenlight banner?

Reply:
Yes, on some older games that haven't updated their store page for some time:
https://store.steampowered.com/app/248190/Knytt_Underground/
https://store.steampowered.com/app/251370/Escape_Goat/

I'll maybe add some other options like the early access info box in a separate PR.